lints/lint_sub_cert_postal_code_prohibited.go

```diff
@@ -16,7 +16,8 @@ func (l *subCertPostalCodeMustNotAppear) CheckApplies(c *x509.Certificate) bool
 }
 
 func (l *subCertPostalCodeMustNotAppear) Execute(c *x509.Certificate) *LintResult {
-	if len(c.Subject.Organization) == 0 || len(c.Subject.GivenName) == 0 || len(c.Subject.Surname) == 0 {
+	// We aren't following the BRs as 7.1.4.2.2 uses "or" and "and" interchangeably when they mean "and".
```

Comment:
Can you change to `BRs 7.1.4.2.2 uses "or" and "and" interchangeably when they mean "and".`